client/a8c-for-agencies/sections/referrals/primary/referrals-overview/layout-body-content.tsx

```diff
@@ -291,11 +291,17 @@ export default function LayoutBodyContent( {
 										: translate( 'Install WooPayments on your clients’ online stores' )
 								}
 								description={
-									isAutomatedReferral
-										? translate( 'Receive a rev share of 0.05% per sale.' )
-										: translate(
-												'Receive a revenue share of 5 basis points (0.05%) on new WooPayments total payments volume (TPV) on clients’ sites.'
-										  )
+									isAutomatedReferral ? (
+										<>
+											{ translate( 'Receive a revenue share of 5 basis points on the' ) }
+											<br />
+											{ translate( 'total payments volume.' ) }
+										</>
+									) : (
```

Comment:
Hi there 👋 . Just wanted to share some i18n feedback about this string. Normally, we want to avoid splitting sentences because it's hard for translators to know if the strings go together. We would recommend putting the  <br /> as an inline tag instead of a separate entity that splits up the sentence. Something like:

`
{ translate(
  <>
    Receive a revenue share of 5 basis points on the
    <br />
    total payments volume.
  </>
) }
`

Reply:
Thanks! Fixed it here: https://github.com/Automattic/wp-calypso/pull/92163